src/Jackett.Common/Indexers/Abnormal.cs

```diff
@@ -170,11 +171,12 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
             await ConfigureIfOK(response.Cookies, response.Cookies.Contains("session="), () =>
             {
                 // Parse error page
-                CQ dom = response.Content;
-                var message = dom[".warning"].Text().Split('.').Reverse().Skip(1).First();
+                var parser = new HtmlParser();
+                var dom = parser.ParseDocument(response.Content);
+                var message = dom.QuerySelector(".warning").TextContent.Split('.').Reverse().Skip(1).First();
```

Comment:
`.Last()` is ok, but change this one. [C#8 added index from end operator](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#indices-and-ranges) like so `array[^n] => array[array.Length-n]`
Reverse().Skip(1).First() takes second from last element, so change as follows:
```suggestion
                var message = dom.QuerySelector(".warning").TextContent.Split('.')[^2];
```

Reply:
C#8 is enabled, but when imported into my VS, it gives an error surrounding this line.

Reply:
Found it. This is incompatible with .net461 because it uses a new type under the hood https://docs.microsoft.com/en-us/dotnet/api/system.index?view=netcore-3.1

Only works with .netcore3.0+ and .netstandard not .netframework

Reply:
@cadatoiva I'm reverting this because it caused this error:
```
Indexers/Abnormal.cs(176,84): error CS0518: Predefined type 'System.Index' is not defined or imported [/home/vsts/work/1/s/src/Jackett.Common/Jackett.Common.csproj]
Indexers/Abnormal.cs(176,84): error CS0656: Missing compiler required member 'System.Index..ctor' [/home/vsts/work/1/s/src/Jackett.Common/Jackett.Common.csproj]
##[error]Error: The process '/opt/hostedtoolcache/dotnet/dotnet' failed with exit code 1
```

Reply:
@ngosang yup. My above comment points out why. net461 is incompatible with this feature. Only once we fully migrate to .Net-Core can we change it.